cni-plugin/main.go

```diff
@@ -237,7 +237,11 @@ func cmdAdd(args *skel.CmdArgs) error {
 				logEntry.Errorf("linkerd-cni: could not create a Firewall Configuration from the options: %v", options)
 				return err
 			}
-			iptables.ConfigureFirewall(*firewallConfiguration)
+
+			if err := iptables.ConfigureFirewall(*firewallConfiguration); err != nil {
```

Comment:
@frankgu968 thanks for the fast turnaround on this PR

The only thing I'd change is to move the `if err !=nil` check to it's own line, to be consistent with the rest of the checks, but that's trivial

Your contribution is much appreciated

Reply:
No problem! I saw the inline delaration pattern from earlier lines as well. Made the changes according to your suggestion @cpretzer

Reply:
thanks @frankgu968